Core/NpcFinder/NpcNameFinder.cs

```diff
@@ -120,9 +120,9 @@ public async Task FindAndClickNpc(int threshold, bool leftClick)
                 {
                     foreach (var location in locFindAndClickNpc)
                     {
-                        var clickPostion = bitmapProvider.DirectBitmap.ToScreenCoordinates(npc.ClickPoint.X + location.X, npc.ClickPoint.Y + location.Y);
+                        var clickPostion = bitmapProvider.DirectBitmap.ToScreenCoordinates(npc.ClickPoint.X + location.X, npc.ClickPoint.Y + location.Y + 30);
                         mouseInput.SetCursorPosition(clickPostion);
-                        await Task.Delay(100);
+                        await Task.Delay(50);
```

Comment:
Are you 100% sure that scanning this quickly the Cursor will catches the desired `CursorClassification` ?

Reply:
No, it was just a test